stream015/src_cpp.cpp

```diff
@@ -8,26 +8,18 @@ int main(int argc, char **argv) {
     int start_num = atoi(argv[1]);
     int count = atoi(argv[2]);
 
-    int list_len = 0;
-    vector<int> list(0); // please use a list in this case kids.
+    vector<int> list;
```

Comment:
You should actually very often prefer std::vector to std::list for performance reasons even though it seems counter intuitive at first.